CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+This page lists the steps needed to set up a development environment and contribute to the project.
+
+1. Fork and clone this repo
+
+2. Install `tox` via `pipx`: https://tox.wiki/en/4.11.3/installation.html#via-pipx
```

Comment:
Use latest instead of concrete version, also make it a markdown link, not explicit. Something like:

[Install tox](https://tox.wiki/en/latest/installation.html#via-pipx).